### PR TITLE
Joyce

### PR DESCRIPTION
**Pull request by JOYCE-CAG**

### CODE REVIEW

**Review by RuthBerhane — COMMENTED** [review]
Can you format your code, git rid of spaces, and add @regression to the AgentModule feature file.

File: src/test/java/com/chroma/features/AgentModule.feature [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Feature: Agent Module Text
+
+  @joyce @AGT-44
```
Comment (safwan216) [inline src/test/java/com/chroma/features/AgentModule.feature:3]: Add the Regression tag here

Comment (arthuroLP) [reply src/test/java/com/chroma/features/AgentModule.feature:3]: guessing this is for another test scenario

**Review by arthuroLP — APPROVED** [review]
try to not have so much space between each line in some of your folders.

**Pull request closed without merging**